tests/py3-ignores.txt

```diff
@@ -27,8 +25,6 @@ tests/test_engine.py
 tests/test_http_cookies.py
 tests/test_logformatter.py
 tests/test_mail.py
-tests/test_pipeline_files.py
-tests/test_pipeline_images.py
```

Comment:
Thanks! I was under the impression until now that headers were to be strings. Good to know.

But different encodings for headers? I would look that up in some RFCs, I can't really imagine that a single response might legitimately have different encodings. (where would that encoding be defined?)